ddl/ddl_worker.go

```diff
@@ -122,42 +132,58 @@ func (d *ddl) checkOwner(t *meta.Meta) (*model.Owner, error) {
 	// the owner will update its owner status every 2 * lease time, so here we use
 	// 4 * lease to check its timeout.
 	maxTimeout := int64(4 * d.lease)
+	if flag == bgJobFlag {
+		// background job doesn't need to guarantee other servers update the schema.
+		maxTimeout = int64(2 * d.lease)
```

Comment:
If u need not guarantee the other servers update the schema, why not using a short time?
`2 * d.lease` is a little long.